tools/clang/unittests/HLSLExec/LongVectors.cpp

```diff
@@ -569,6 +569,23 @@ void runAndVerify(ID3D12Device *D3DDevice, bool VerboseLogging,
                                 ValidationConfig.Type, VerboseLogging));
 }
 
+template <typename IN_TYPE, typename OUT_TYPE>
+static std::vector<OUT_TYPE> TransformVector_1(
+    const InputSets<IN_TYPE> &Inputs,
```

Comment:
I also think it would be good to add a short comment to this helper just explaining that the store the output parameters in the second half of the output buffer to keep things simple as the significant majority of our tests only need one output buffer. So, we stick with that pattern.